documentation/index.rst

```diff
@@ -452,6 +452,30 @@ string data in all Python versions.
    :meth:`py3:str.encode`
 
 
+.. function:: str_to_binary(s, encoding='utf-8', errors='strict')
+
+   Encodes s to :data:`binary_type` in python 3 only. No-op in python 2.
+   *encoding*, *errors* are same as :meth:`py3.str.encode`
```

Comment:
For completeness, I'd mention the type of the argument (it's possible to infer it from the rest but it took me a while). And do this for all functions.